tests/apis/rootPath.json

```diff
@@ -0,0 +1,41 @@
+{
+    "swagger": "2.0",
```

Comment:
Hm, are we even using the ones in this folder?

Reply:
The `generation.js` is a test itself - it uses the files in `tests/apis/` folder and check if the result of code generation is a string and if generated typescript code compiles. Then probably the new file is not needed in this folder as it doesn't provide any value, do you agree?

Reply:
agree!